shared/bin/setup-cephadm.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+refresh=0
+
+for i in "$@"; do
+    case $i in
+    --refresh)
+        refresh=1
+    ;;
+    esac
+done
+
+# Make sure the host SSH directory is mapped into the Docker container.
+if [ ! -e "/root/.ssh/id_rsa" -o ! -e "/root/.ssh/id_rsa.pub" ]; then
+    echo "ERROR: The host SSH directory is not mapped into the Docker container. Make sure to use the command line argument '-v ~/.ssh:/root/.ssh:ro' when starting the Docker container."
+    exit 1
+fi
```

Comment:
I went another approach: modify the key paths in ssh-config on host first.
```
# change to cephadm module folder first
sed -i "s,$(pwd),/ceph/src/pybind/mgr/cephadm," ssh-config
```
In this case I don't need to bind mount key folder to container.